### PR TITLE
add a tower.yml

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR proposes a tower.yml to report the project-level outputs in Tower. Hopefully I haven't missed some mechanism that can already do this. 

Will close https://github.com/nf-core/fetchngs/issues/121

I don't know how to solve the issue in the CI, but I don't think it's related to these changes.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/fetchngs/tree/master/.github/CONTRIBUTING.md)
  - [x] If necessary, also make a PR on the nf-core/fetchngs _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: tower.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "samplesheet/multiqc_config.yml":
+    display: "MultiQC config file that can be passed to most nf-core pipelines"
+  "metadata/*.runinfo_ftp.tsv":
+    display: "Re-formatted metadata file downloaded from the ENA"
+  "metadata/*.runinfo.tsv":
+    display: "Original metadata file downloaded from the ENA"
+  "metadata/*.metadata.txt":
+    display: "Original metadata file generated using the synapse show command"
+  "metadata/*.list.txt":
+    display: "Original output of the synapse list command"
```
Comment (drpatelh) [inline tower.yml:15]:
```suggestion
  samplesheet.csv:
    display: "Auto-created samplesheet with collated metadata and FASTQ paths"
  id_mappings.csv:
    display: "File with database identifier mappings that can be used to rename samples"
  multiqc_config.yml:
    display: "MultiQC config file for bulk renaming of sample names from database ids"
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ This downloads a text file called `SRR_Acc_List.txt` that can be directly provid
 
 ### Enhancements & fixes
 
+- [#121](https://github.com/nf-core/fetchngs/pull/123) - Add Tower reporting
```
Comment (drpatelh) [inline CHANGELOG.md:28]:
```suggestion
- [#121](https://github.com/nf-core/fetchngs/pull/123) - Add `tower.yml` to render Reports in Tower
```

File: tower.yml (hunk 1 above)
Comment (drpatelh) [reply tower.yml:15]: Thanks @pinin4fjords ! I wonder whether it is worth keeping the Reports simple to just the main outputs. For example, not sure it's worth having a `*.runinfo.tsv` for all samples rendered as a report especially if you are downloading loads of them.

Should probably test the final `tower.yml` to confirm.

**Comment by pinin4fjords** [comment]
Thanks @drpatelh - I'll test in Tower and report back

**Comment by pinin4fjords** [comment]
@drpatelh I can confirm this has the desired impact in Tower, but I needed the switch to CSV too. Do you want me to leave that change here and fix the test data, or discuss separately?

**Comment by drpatelh** [comment]
Awesome!

> Do you want me to leave that change here and fix the test data, or discuss separately?

I have added a test-dataset with the `.csv` extension now. We use ".txt" quite a bit in the pipeline repo that will all need to be replaced with ".csv". Can do that here if you like too?

**Comment by pinin4fjords** [comment]
Okay, I think this should do it @drpatelh (tests pending)!

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -10,61 +10,30 @@ on:
...
 env:
   NXF_ANSI_LOG: false
-  CAPSULE_LOG: none
 
 jobs:
   test:
     name: Run pipeline with test data
     # Only run on push if this is the nf-core dev branch (merged PRs)
-    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/fetchngs') }}"
+    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/differentialabundance') }}"
```
Comment (drpatelh) [inline .github/workflows/ci.yml:18]:
```suggestion
    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/fetchngs') }}"
```

Comment (pinin4fjords) [reply .github/workflows/ci.yml:18]: Oops, sorry, copy/paste probs!

**Review by drpatelh — APPROVED** [review]
Thanks @pinin4fjords ! I will merge this in and fix any conflicts in https://github.com/nf-core/fetchngs/pull/115

**Pull request merged**